src/views/home/sections/MentorsVolunteers/MentorsVolunteers.tsx

```diff
@@ -29,7 +29,8 @@ function MentorsVolunteers() {
 							</p>
 							<Button
 								variant="museum"
-								href="https://forms.gle/tcnikpj5gHnGPNvx7"
+								href="/mentor"
+								role=""
```

Comment:
Clarification: The default value for `role` is `button` right? Is that why it's set to an empty string here?

Reply:
Yeah, in this case, the actual wording of the button text would be more appropriate for a link (default role for `<a>`). This would not be necessary for the "Apply as ..." buttons in the Home landing since those are worded to be "actionable".